src/PrestaShopBundle/Resources/config/services/core/grid/grid_position_definition_factory.yml

```diff
@@ -57,3 +57,10 @@ services:
       $positionField: 'position'
       $parentIdField: 'id_category'
       $firstPosition: 1
+
+  prestashop.core.grid.feature.position_definition:
```

Comment:
fqcn here

Reply:
right :+1:

Reply:
@FabienPapet we can't use FQCN here the class is shared among multiple services, so each one neds its dedicated name since none is more default than another